openapi/resource-server.yaml

```diff
@@ -291,27 +291,60 @@ paths:
                   quoteId: 'https://ilp.rafiki.money/quotes/ab03296b-0c8b-4776-b94e-7ee27d868d4d'
                   metadata:
                     externalRef: INV2022-02-0137
+              Create an outgoing payment based on an incoming payment:
+                value:
+                  walletAddress: 'https://ilp.rafiki.money/alice/'
+                  incomingPaymentId: 'https://ilp.rafiki.money/incoming-payments/8d4e4776-2e55-4e5a-bcbe-8348ed1e86de'
+                  debitAmount:
+                    value: '2500'
+                    assetCode: USD
+                    assetScale: 2
+                  metadata:
+                    externalRef: INV2022-02-0137
             schema:
-              type: object
-              properties:
-                walletAddress:
-                  $ref: ./schemas.yaml#/components/schemas/walletAddress
-                quoteId:
-                  type: string
-                  format: uri
-                  description: The URL of the quote defining this payment's amounts.
-                metadata:
-                  type: object
-                  additionalProperties: true
-                  description: Additional metadata associated with the outgoing payment. (Optional)
-              required:
-                - quoteId
-                - walletAddress
-              additionalProperties: false
+              oneOf:
+                - type: object
+                  properties:
+                    walletAddress:
+                      $ref: ./schemas.yaml#/components/schemas/walletAddress
+                    quoteId:
```

Comment:
I think in that vain, we should probably also rename that to just be `quote`?

Reply:
I was going to rename this to `quote` to make this endpoint consistent internally but this will be harder than I initially realized. We are currently tightly coupled to the [open payments snippets repo](https://github.com/interledger/open-payments-snippets) via [these links in our `<ChunkedSnippets>` in the docs](https://github.com/interledger/open-payments/blob/main/docs/src/content/docs/guides/make-recurring-payments.mdx?plain=1#L168-L180).

This means if we rename to `quote`, the docs will be wrong until we update open payments snippets. Not great really. I think we have two options:

- not rely on the open payments snippets in our docs 

OR

- if we want to depend on the open payments snippets, make the snippets a package in this repository

Just to be clear, the snippets reflect whatever version of the open-payments package it has as a dependency. That can be behind the latest. That's fine. It's those references in the docs which assume the Open Payments snippets are up to date with the latest changes here which is the problem.

Reply:
To be clear I don't intend on tackling any of this here - just explaining why I opted not to rename to `quote`. If we make any changes along these lines we can do so in another task.